<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NODE167" name="Test" tests="1" failures="1" timestamp="23 Aug 2016 11:02:39 GMT" time="47.333" errors="0">
  <testcase name="DomesticTravel" time="27.539" classname="com.gspann.selenium.IrctcTest">
    <failure type="java.lang.AssertionError" message="expected [69] but found [67]">
      <![CDATA[java.lang.AssertionError: expected [69] but found [67]
	at com.gspann.selenium.IrctcTest.DomesticTravel(IrctcTest.java:97)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- DomesticTravel -->
</testsuite> <!-- Test -->
